web/static/js/reducers/questionnaire.js

```diff
@@ -222,26 +238,47 @@ const changeStepStore = (state, action) => {
 const addStep = (state, action) => {
   return [
     ...state,
-    newStep()
+    newMultipleChoiceStep()
   ]
 }
 
-const newStep = () => ({
-  id: uuid.v4(),
-  type: 'multiple-choice',
-  title: '',
-  store: '',
-  prompt: {
-    'en': {
-      sms: '',
-      ivr: {
-        text: '',
-        audioSource: 'tts'
+const newLanguageSelectionStep = (first: string, second: string): LanguageSelectionStep => {
```

Comment:
I had to remove `newStep` and introduce some code duplication to these factory methods for each kind of step, because I couldn't find a DRY way that would please Flow. For some (I guess good) reason Flow can't infer what happens when you use `Object.assign` so any attempt of merging "common step properties" with specific ones utterly failed. I suggest not to touch this for the time being. It works ok, Flow likes it and it's easy to read and hard to mess up. There are much more urgent issues to address :).